pages/escape.md

```diff
@@ -0,0 +1,26 @@
+---
+title: ESCAPE
+---
+
+The [ESCAPE](https://projectescape.eu/) science cluster brings together communities and research infrastructures in the areas of astronomy and accelerator-based particle physics, specifically ESFRI facilities (SKAO, CTAO, KM3NeT, EST, ELT, HL-LHC, FAIR) and pan-European research infrastructures (CERN, ESO, JIVE). 
+
+Software applications for gathering and simulating the scientific data, in the order or exa-bytes, in ESCAPE span code bases of several million lines of source code and are written by researchers and engineers in various compiled (eg. C++, Cuda) and interpreted (eg. Python) languages. With experiments spanning over multiple decades the quality of the software developed and used is imperative for the success of the scientific endeavors in view of efficient computing and code maintainability.
+
+## Main goals
+
+- Improve access to data and tools to unlock innovation for the society at large.
+- Provide data with FAIR principles to increase researchers’ efficiency thanks to scientific data interoperability and establish new methodological approaches and rules for quality certified data and science tools sharing.
+- Build a European cross-border and multi-disciplinary open innovation environment for research data, knowledge and services, while connecting EOSC and ESFRI.
+- Facilitate interdisciplinary and networked research between different sciences, through research infrastructure ecosystem and by supporting data publishing, analytics, computational capacity, virtual analysis environments and workflow systems.
+- Create of economies of scale, through the adoption of common approaches for data management
+- Educate and train the scientific and wider user communities, to ensure the up-take of ESCAPE’s results.
+
+## Services
+
+### Software catalogue
+
+The [Open-source Scientific Software and Service Repository](https://escape-ossr.gitlab.io/ossr-pages/) enables you to save time and resources by improving the reproducibility of scientific results through high-quality software from within the community.
```

Comment:
The sudden switch to "you" is very odd here. I would say "enables researchers...".